api/models/GroupExtension.js

```diff
@@ -0,0 +1,14 @@
+module.exports = bookshelf.Model.extend(Object.assign({
+  tableName: 'group_extensions',
+  requireFetch: false,
+  hasTimestamps: true,
+  group () {
```

Comment:
prefer lines between function definitions

Reply:
The linter complains about this, because these function defs are in an object

Reply:
hmm, we do this everywhere though, we should probably update our linter definitions